.github/workflows/ci.yml

```diff
@@ -51,3 +51,18 @@ jobs:
 
       - name: Build SATySFi
         run: opam exec -- make all
+
+      - name: Run tests
+        run: |
+          opam exec -- dune runtest
+
+      - name: Install SATySFi
+        run: |
+          opam install -y .
+          ./download-fonts.sh
+          sudo ./install-libs.sh
```

Comment:
This is a matter of taste.  You can install the lib files under `~/.satysfi` instead.

```
./install-libs.sh ~/.satysfi
```

Reply:
fair enough!